### ISSUE TITLE
Text when loading world is duplicated.

### ISSUE THREAD

**Issue by seriousfreezing** [juancarloscp52/BedrockIfy#379]
Every time I load a world, it shows the modified screen of the mod itself and a text about the loading information, but then a duplicate text of the same information appears a little further up.

I was wondering if there was a fix for this. I think I'd like to have some way to change the color of this text to better fit a dark mode resourcepack.
![Captura de tela 2024-09-05 174042](https://github.com/user-attachments/assets/ebacf82c-3cb1-49a8-93fc-a9384d39f47c)

https://paste.atlauncher.com/view/64851bf5-8612-4ed1-9bb7-84d790519774

**Comment by juancarloscp52** [comment]
Sorry for the delay lol, is the issue still happening?

**Comment by seriousfreezing** [comment]
Yes, but I don't know if this is due to compatibility with some other mod.

**Comment by juancarloscp52** [comment]
If possible, please include a list of mods (previous log seems to have
expired) and I'll try to take a look at what is causing it.

Thank you :)

**Comment by seriousfreezing** [comment]
https://modrinth.com/modpack/bedrock-reloaded/version/25.0.0
Thank you, I've updated your mod too and the effect is still there.
In order to test it, you have to remove the “Bedrocking.zip” resourcepack.

**Comment by juancarloscp52** [comment]
Okey, This is caused by Fancy Menu. My recommendation is to create a new Layout for the Receiving Level Screen and remove the loading terrain text.

**Comment by seriousfreezing** [comment]
Currently, it seems that if you remove this text in the new versions of FancyMenu, it will also remove the Bedrockify frame decoration.
